src/state/nftMarket/types.ts

```diff
@@ -200,7 +200,7 @@ export interface ApiCollection {
 
 // Get all collections
 // ${API_NFT}/collections/
-export interface ApiCollectionsReponse {
+export interface ApiCollectionsResponse {
```

Comment:
Looks like this type never used
Let's reference it into https://github.com/pancakeswap/pancake-frontend/blob/develop/src/state/nftMarket/helpers.ts#L37

Reply:
Done